client/packages/host/src/components/Android.tsx

```diff
@@ -23,6 +23,15 @@ import { useTranslation } from '@common/intl';
 import Viewport from './Viewport';
 import { LoginIcon } from './Login/LoginIcon';
 
+const DEFAULT_LOCAL_SERVER = {
```

Comment:
this is because I had an issue - the dnssd was returning no servers, however the server was running on the tablet and I was in Server mode.
Because no servers were returned by dnssd, the loading spinner was shown infinitely.

Doing this is safe enough: the defaults are hard coded in the android code, and if the server is unreachable, an error is shown to the user.